src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &--4 {
+    display: flex;
+    align-items: center;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be a space before the opening brace `{`. Please add a space before the `{` on line 15 to maintain consistency.